src/amo/components/ReportAbuseButton/index.js

```diff
@@ -81,13 +82,26 @@ export class ReportAbuseButtonBase extends React.Component<InternalProps> {
   };
 
   onReportButtonClick = async (event: SyntheticEvent<any>) => {
+    const { _hasAbuseReportPanelEnabled, addon, dispatch } = this.props;
+
     event.preventDefault();
 
-    const { _hasAbuseReportPanelEnabled, addon, dispatch } = this.props;
-    if (_hasAbuseReportPanelEnabled()) {
+    if (
+      _hasAbuseReportPanelEnabled() &&
+      // The integrated abuse report panel currently support only extension and
+      // themes (e.g. we do only have abuse categories and related localized
+      // descriptions only for these two kind of addons), and so currently it
+      // is going to refuse to create an abuse report panel for langpacks,
+      // dictionaries and search tools.
+      //
+      // Static themes should be supported but there is a bug in FF, see:
+      // https://github.com/mozilla/addons-frontend/issues/8762#issuecomment-553430081
```

Comment:
We should file a follow-up to add static themes to this list once the FF bug is fixed.

Reply:
Good call, thanks. Done here: https://github.com/mozilla/addons-frontend/issues/8931